### ISSUE TITLE
WPCS config

### ISSUE THREAD

**Issue by GaryJones** [humanmade/whitelist-html#2]
Suggestion for the README - add a section on how devs can amend their `phpcs.xml` (or `ruleset.xml`) file to allow `whitelist_html()` as a valid escaping function when checking PHP_CodeSniffer + WordPress Coding Standards sniffs.

I'm sure @JDGrimes can give you the correct few lines it would need, but I vaguely think it would be something like:

```xml
<rule ref="WordPress.XSS.EscapeOutput">
	<properties>
		<property type="array" name="customAutoEscapedFunctions" value="whitelist_html"/>
	</properties>
</rule>
```

Otherwise, WPCS is going to be complaining every time `whitelist_html()` is used.

**Comment by JDGrimes** [comment]
Yes, that looks right. Although I think maybe that `print_whitelist_html()` doesn't need to be added to the printing function list, since it automatically escapes its output.
